test/test.oauth2.ts

```diff
@@ -44,8 +44,12 @@ describe('OAuth2 client', () => {
       response_type: 'code token'
     };
 
-    const oauth2client =
-        new OAuth2Client(CLIENT_ID, CLIENT_SECRET, REDIRECT_URI);
+    const oauth2client = new OAuth2Client({
```

Comment:
likewise.

Reply:
The first test uses the object format, the others use the exploded format.  I also added a test that overrides authbaseUrl and tokenUrl - because no test covered those at all :(